src/analysis/calculus/cont_diff_def.lean

```diff
@@ -0,0 +1,1597 @@
+/-
+Copyright (c) 2019 Sébastien Gouëzel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sébastien Gouëzel
+-/
+import analysis.calculus.mean_value
+import analysis.normed_space.finite_dimension
+import analysis.normed_space.multilinear
+import analysis.calculus.formal_multilinear_series
+import data.enat.basic
+import tactic.congrm
```

Comment:
Has the split made any of these no longer necessary?

Reply:
Yes, indeed. Fixed.